app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,9 @@
+class PostsController < ApplicationController
+  def posts
+    @posts = Post.all
+  end
```

Comment:
- [ ] Please make sure that you are using the correct `convention` for development in `ruby on rails`, in rails your endpoint should be one of the standard `web` endpoints to access, update and search or create the resources using, `get`, `show`, `new`, `create`, `update` or `delete`. Anything apart from this is is considered the `method` useful for doing some operation. Based on these endpoints, Ruby will automatically look for the `views too send or receive the request or response from or to the `user`. Your code snippet should look like below: 

``` suggestion
  def index
    @posts = Post.all
  end
```